README.md

```diff
@@ -146,6 +146,7 @@
     - [Manage the Task](tools/dm/manage-task.md)
     - [Cluster Operations](tools/dm/cluster-operations.md)
     - [Troubleshoot](tools/dm/troubleshooting.md)
+    - [Benchmark](tools/dm/benchmark.md)
```

Comment:
I double checked with @GregoryIan. It's unnecessary to add the DM benchmark file to the README.md (which will be displayed on the PingCAP website), so please remove it here.